src/TMainConsole.h

```diff
@@ -0,0 +1,160 @@
+#ifndef MUDLET_TMAINCONSOLE_H
+#define MUDLET_TMAINCONSOLE_H
+
+/***************************************************************************
+ *   Copyright (C) 2008-2012 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2014-2016, 2018-2020 by Stephen Lyons                   *
+ *                                               - slysven@virginmedia.com *
+ *   Copyright (C) 2016 by Ian Adkins - ieadkins@gmail.com                 *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+
+#include "TBuffer.h"
+#include "TConsole.h"
```

Comment:
Wrong include - should be `#include "TMainConsole.h"`! And the `TBuffer.h` isn't required as it is the `TConsole.h` file...

Reply:
Huh? How should TMainConsole.h include itself?

`TBuffer.h` is correct, that can be dropped, and I suppose I can also replace the `TConsole.h` include with an empty `class TConsole` declaration, but as about any user of TMainConsole is going to access some TConsole methods or data anyway I don't think that's helpful.

Reply:
🤦‍♂️ Ah I may have mistaken the `.h` for the `.cpp` file...